src/components/families/family-sidebar/FamilySidebar.tsx

```diff
@@ -163,7 +175,19 @@ const FamilySidebar = ({ family, isOpen, onClose, onEditFamily }: Props) => {
         <Typography variant="h2">
           {family.parent.first_name} {family.parent.last_name}
         </Typography>
-        <Divider variant="fullWidth" />
+
+        <Typography variant="h3" className={classes.heading}>
```

Comment:
I just noticed classes.heading has font size 16 here, but in FamilySidebarForm it's classes.heading has font size 18. These sizes should be the same according to the designs. This isn't related to the PR, but maybe we can add it to a ticket to address design review.

Reply:
yeah you're right, this is pending a design clarification since we swap between 16/18 a bit 😅 will make a ticket when we sync